sdk/streamlib-python-wheel/tests/texture_ring_producer_probes.py

```
@@ -0,0 +1,135 @@
+the other probes use, tagged with `probe` because a scenario runs two of them.
+"""
+
+import json
+import os
+import traceback
+
+from streamlib import (
+    ProcessorOutputTextureRing,
+    VideoFrame,
+    clock,
+    input,
```

Ruff (0.16.2)

[error] Import `input` is shadowing a Python builtin

(A004)